.editorconfig

```diff
@@ -0,0 +1,654 @@
+root = true
```

Comment:
This file contains a lot of defaults, doesn't it?

Reply:
Better that there are default settings and we change them along the way.
Than none at all.